spec/namespaces/security.yaml

```diff
@@ -1992,6 +2047,36 @@ components:
         type: string
         description: Action group to retrieve.
       required: true
+    security.get_node_certificates::path.node_id:
+      name: node_id
+      in: path
+      schema:
+        type: string
+      required: true
+    security.get_all_certificates::query.cert_type:
+      name: cert_type
+      in: query
+      schema:
+        type: string
+      required: false
+    security.get_node_certificates::query.cert_type:
+      name: cert_type
+      in: query
+      schema:
+        type: string
+      required: false
+    security.get_all_certificates::query.timeout:
+      name: timeout
+      in: query
+      schema:
+        type: string
```

Comment:
Are these strings or one of the duration types?

Reply:
it is expected as a string to be utilized here: https://github.com/opensearch-project/security/blob/cc26b94ca84b562ae75ed8e2958a343831682042/src/main/java/org/opensearch/security/dlic/rest/api/CertificatesApiAction.java#L89

Reply:
I mean should this be 

```
      schema:
        $ref: '../schemas/_common.yaml#/components/schemas/Duration'
```

Reply:
i see, I have replaced it.